avocado/utils/cpu.py

```diff
@@ -51,6 +53,20 @@ def _get_cpu_info():
     return cpuinfo
 
 
+def _get_cpu_status(cpu):
+    """
+    Check if a CPU is online or offline
+
+    :pram cpu: CPU number 1 or 2 or 39
+    :type cpu: integer
+    :returns: `bool` True if online or False if not
+    :rtype: 'bool'
+    """
+    if '1' in open('/sys/devices/system/cpu/cpu%d/online' % int(cpu)).read():
```

Comment:
I think you can trust people to use valid inputs and you can simply use `cpu%s/online' % cpu)`.